### PR TITLE
Fixed autoupdate for m2m relation.

### PR DESCRIPTION
**Pull request by normanjaeckel**

### CODE REVIEW

**Comment by emanuelschuetze** [comment]
@FinnStutzenstein Is this ok for you? Can we merge this?  I don't know how to test it exactly.

**Comment by FinnStutzenstein** [comment]
Also please rebase.

File: openslides/mediafiles/views.py [hunk 1]
```diff
@@ -75,6 +77,8 @@ def destroy(self, request, *args, **kwargs):
         # unattended side effects so be careful especially when accessing files
         # on server via Django methods (file, open(), save(), ...).
         mediafile = self.get_object()
+        using = router.db_for_write(Mediafile, instance=mediafile)
+        pre_delete.send(sender=Mediafile, instance=mediafile, using=using)
```
[the remark this answers is not in the document]
Comment (normanjaeckel) [reply openslides/mediafiles/views.py]: Can remember why I added these lines. Removed.

**Comment by normanjaeckel** [comment]
All done.

**Comment by FinnStutzenstein** [comment]
@normanjaeckel I tested again. It works as expected. I'll merge this now.

**Comment by emanuelschuetze** [comment]
Rebase to one commit would be fine...

**Pull request merged**